<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1068</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1068</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="OutPhotoLab">
    <property name="geometry">
     <rect>
      <x>704</x>
      <y>11</y>
      <width>353</width>
      <height>500</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>353</width>
      <height>500</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>OutPut</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>11</y>
      <width>353</width>
      <height>500</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>353</width>
      <height>500</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="CharacterSpaceTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="CharacterSpaceLab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Character space</string>
    </property>
   </widget>
   <widget class="QLabel" name="CharacterSizeLab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Character size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CharacterSizeTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="PenWidthLab">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pen width</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PenWidthTxt">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>32</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CharacterFamilyTxt">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>liguofu</string>
    </property>
   </widget>
   <widget class="QLabel" name="CharacterFamilyLab">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>128</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Character Family</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1068</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="Initial"/>
    <addaction name="Generate"/>
    <addaction name="ShowImg"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Initial">
   <property name="text">
    <string>Initial(&amp;I)</string>
   </property>
   <property name="toolTip">
    <string>Initial Text</string>
   </property>
  </action>
  <action name="Generate">
   <property name="text">
    <string>Generate(&amp;G)</string>
   </property>
   <property name="toolTip">
    <string>Generate Photo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="ShowImg">
   <property name="text">
    <string>Show img(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
